archivers/7zip/Portfile

```diff
@@ -53,10 +53,15 @@ post-patch {
     reinplace "s|LFLAGS_ALL = $\(\LFLAGS_STRIP\)\|LFLAGS_ALL =|" \
     ${worksrcpath}/C/7zip_gcc_c.mak \
     ${worksrcpath}/CPP/7zip/7zip_gcc.mak
+    # do not force -O2 flag during compilation
+    reinplace "s|CFLAGS_BASE = $\(\MY_ARCH_2\)\ -O2|CFLAGS_BASE = $\(\MY_ARCH_2\)\ -O2 ${configure.cflags}|" \
```

Comment:
I don't get why there are backslashes in front of the Ms and the spaces.

Reply:
As I wrote above, without those backslashes it gives a "no such variable" error (both locally and CI).

Reply:
Please figure out why. "M" is not a special character etc. Something weird is happening and mysteries are bad.